language-management/src/js/views/modalView.js

```diff
@@ -0,0 +1,30 @@
+import Event from "../events/event";
+
+class ModalView {
+  constructor() {
+    this.openEvent = new Event();
+    this.closeEvent = new Event();
+
+    this.modalEl = document.querySelector(".modal");
+    this.overlayEl = document.querySelector(".overlay");
+    this.addBtnEl = document.querySelector(".features__add");
+    this.cancelBtnEl = document.querySelector(".modal__top__btn");
+
+    this.addBtnEl.addEventListener("click", () => {
```

Comment:
Should create a separate method for the callback:
```
() => {
      this.modalEl.classList.add("modal--active");
      this.overlayEl.classList.add("overlay--active");
      this.openEvent.trigger();
    }
    ```
    call it like this:  `this.addBtnEl.addEventListener("click", this.handleOpenAddModal)`